README.md

```diff
@@ -1 +1,6 @@
-# openvino_quantization
\ No newline at end of file
+# openvino_quantization
+
+Link to find ResNet_V2_101 model:
+https://www.tensorflow.org/lite/guide/hosted_models
+Direct link to download this model:
+https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz
```

Comment:
Please add more information here - what are script files for, how to run them. Add estimated accuracy